pkg/trace/writer/sender.go

```diff
@@ -319,9 +321,6 @@ func (s *sender) recordEvent(t eventType, data *eventData) {
 	s.cfg.recorder.recordEvent(t, data)
 }
 
-// userAgent is the computed user agent we'll use when communicating with Datadog
-var userAgent = fmt.Sprintf("Datadog Trace Agent/%s/%s", info.Version, info.GitCommit)
```

Comment:
Nice. I was actually pondering how to solve this. I like your solution.

Reply:
Thanks! Yeah it seems a bit odd to have this as a package variable when there's a nice config structure it could go into